admin-dev/themes/default/package-lock.json

```diff
@@ -3558,6 +3569,33 @@
         "css-tree": "^1.1.2"
       }
     },
+    "csso-webpack-plugin": {
+      "version": "2.0.0-beta.3",
```

Comment:
Is this OK to use a beta package?

Reply:
Wrong naming I think so, it's never been stable, but it's just a wrapper over CSSO, I don't even know how it can be beta